boards/DCM/Inc/Io/Io_Imu_LSM6DS33.h

```diff
@@ -0,0 +1,39 @@
+#ifndef IO_IMU_LSM6DS33
+#define IO_IMU_LSM6DS33
```

Comment:
```suggestion
#pragma once
```

Reply:
This isn't required anymore for our current build

Reply:
done